Sources/SnapshotTesting/Attachment.swift

```diff
@@ -5,9 +5,13 @@ public struct Attachment {
   internal let rawValue: XCTAttachment
   #endif
 
-  init(string: String, name: String? = nil) {
+  init(string: String, name: String? = nil, uniformTypeIdentifier: String? = nil) {
```

Comment:
This lets us tag diff artifacts with types...though turns out that QuickLook doesn't have a nice rendering for diffs without a plugin :(